include/a3/ui_f/hpp/defineDIKCodes.inc

```diff
@@ -186,5 +186,4 @@
 #define INPUT_ALT_OFFSET	2048
 
 
-#endif /* DIK_ESCAPE */
```

Comment:
Would prefer fixing the non-conforming tools.
Instead of editing the conforming files to support a broken tool.

Reply:
It will be fixed. No reason to stop this PR from getting merging.